bin/MadGraph5_aMCatNLO/cards/production/13TeV/Singlet_Triplet_Higgs_portal_DM_ms200_mt220_ct3mm/Singlet_Triplet_Higgs_portal_DM_customizecards.dat

```diff
@@ -0,0 +1,9 @@
+set param_card frblock 1 200
+set param_card frblock 2 220
+set param_card frblock 3 7.0e-09
+set param_card mass 9000005 200
+set param_card mass 9000006 220
+set param_card mass 9000007 220.16
+set param_card decay 9000005 0.0
+set param_card decay 9000006 7.19464e-20
+set param_card decay 9000007 6.6e-14
```

Comment:
The decay width determines the lifetime of the particle. When we faced this issue initially, we contacted the madgraph community to help us. The thread is linked below. [1] The short summary is that since there is no other resonant diagram that interferes with the process, the scaling treatment of small width is not affected. The only change in including the small_width_treatment parameter will it will get rid of the error message but that is just an outcome of the print statement. With respect to the underlying mechanism of Madgraph, there is no change. Please let me know if you have a better suggestion for me.

[1] https://answers.launchpad.net/mg5amcnlo/+question/680594

Reply:
Yes indeed the scaling treatment won't really cause any problem for your case when no other resonance interferes. I was trying to explain that what you set doesn't really have any meaning for MadGraph since it doesn't use the value you set at all (use the fake width as Olivier stated), and was worried since the decay widths was set set for some reasons. But if it's not and you are also fine with the NWA it's also fine for me as well.